js/main.js

```diff
@@ -39,62 +51,65 @@ async function initCamera() {
  * all processing steps enabled in current state.
  */
 function processKeyPoints(combinedKeyPoints) {
-  if (combinedKeyPoints == undefined || combinedKeyPoints.length != 2) 
-    throw "Expected 2 key point arrays, but received " + combinedKeyPoints;
+  if (combinedKeyPoints === undefined || combinedKeyPoints.length !== 2) {
+    throw Error(
+      `Expected 2 key point arrays, but received ${combinedKeyPoints}`
+    );
+  }
 
   const facePoints = combinedKeyPoints[0];
   const handPoints = combinedKeyPoints[1];
 
   if (state.isCollectionOn) {
-    const collectionSize = data.collectFeatures(facePoints, handPoints, state.label)
-    ui.updateCollectionText(collectionSize);
+    const collectionSize = collectFeatures(facePoints, handPoints, state.label);
+    updateCollectionText(collectionSize);
   }
 
   if (state.isInferenceOn) {
-    if (facePoints != undefined && handPoints != undefined) {
-      models.computeInference(facePoints, handPoints)
-        .then((inference) => ui.updateInferenceText(inference[0]));
+    if (facePoints !== undefined && handPoints !== undefined) {
+      computeInference(facePoints, handPoints).then((inference) =>
+        updateInferenceText(inference[0])
+      );
     }
   }
 }
 
 /*
- * Each call to update carries out all key point computations, 
- * any enabled processing, and draws the next frame. If no errors 
+ * Each call to update carries out all key point computations,
+ * any enabled processing, and draws the next frame. If no errors
  * occur, asks the runtime to be called again on next frame update.
  */
 async function startEngine(canvas, video) {
   (function update() {
-    models.computeCombinedKeyPoints(video)
-      .then((combinedFeatures) => draw.frame(canvas, video, combinedFeatures))
+    computeCombinedKeyPoints(video)
+      .then((combinedFeatures) => frame(canvas, video, combinedFeatures))
       .then((combinedKeyPoints) => processKeyPoints(combinedKeyPoints))
       .then(() => requestAnimationFrame(update))
-      .catch((err) => console.error("Could not compute and render frame: ", err));
-  })()
+      .catch((err) =>
+        // eslint-disable-next-line no-console
+        console.error('Could not compute and render frame: ', err)
+      );
+  })();
 }
 
 /*
  * Initializes video feed models and tf.js runtime.
  * Failures should be fatal.
  */
 async function initialize() {
-  return Promise.all([
-    initCamera(),
-    models.initialize()  
-  ]);
+  return Promise.all([initCamera(), initializeModel()]);
 }
 
 /* Initializes necessary components and starts the
  * inference engine.
  */
 async function main() {
-
   await initialize();
 
-  ui.setState(state);
-  ui.initButtons();
-  ui.initVideo();
-  const canvas = ui.initCanvas()
+  setStateUI(state);
+  initButtonsUI(exportData);
+  initVideoUI();
+  const canvas = initCanvas();
 
   startEngine(canvas, video);
```

Comment:
Not really related to this PR, but ESLint complains that video isn't defined ("'video' is not defined  no-undef") so I think we could fix that in this PR alongside other ESLint errors. What do you think?

I looked at the code and I believe it was meant to be `state.video` instead of `video` (@misterpeddy correct me if I'm wrong).

Reply:
Great, thank you!!